### DISCUSSION TITLE
PBM negotiated rates

### DISCUSSION THREAD

**Discussion by ballengeec68** [CMSgov/price-transparency-guide#486] [Q&A] [answered]
As a PBM, for the negotiation rate, are you looking for it through a pharmacy breakdown or a general drug NDC breakdown?

**Comment by shaselton-usds** [comment, marked as answer]
Are you asking in the context of the pharmacy machine readable file? If so, enforcement on that file is being deferred. Otherwise, the in-network file requires pharmacy information for bundles or other possible APMs and the individual negotiated rates for the bundle/APM isn't required for each item.

**Reply by shaselton-usds** [reply]
closing this thread.
